gusto/core/configuration.py

```diff
@@ -202,6 +202,7 @@ class SUPGOptions(WrapperOptions):
     tau = None
     default = 1/sqrt(15)
     ibp = IntegrateByParts.TWICE
+    suboptions = None # Dictionary containing field_name and term_labels
```

Comment:
As well as fixing the lint, can you expand on this comment just to make it a bit clearer what `suboptions` should be, i.e. the keys of the dictionary should be strings for the names of the fields, and the values should be a list of terms to apply the SUPG test function to